exporter/clickhousetracesexporter/clickhouse_exporter.go

```diff
@@ -346,8 +347,8 @@ func newStructuredSpan(otelSpan ptrace.Span, ServiceName string, resource pcommo
 		ResourceTagsMap:   resourceAttrs,
 		HasError:          false,
 		TraceModel: TraceModel{
-			TraceId:           otelSpan.TraceID().HexString(),
-			SpanId:            otelSpan.SpanID().HexString(),
+			TraceId:           utils.GetTraceId(otelSpan.TraceID()),
```

Comment:
@nityanandagohain We can use [traceUtil](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2d65254ce3f42dd11934aa3f33ae77f242b0ed20/internal/coreinternal/traceutil/traceutil.go#L62) from contrib rather than creating our own utils.

Reply:
It's an internal package you won't be able to import it.

Reply:
Got it